oshi-core/src/main/java/oshi/util/FileUtil.java

```diff
@@ -69,7 +69,7 @@ private FileUtil() {
      *         list if file could not be read or is empty
      */
     public static List<String> readFile(String filename, boolean reportError) {
-        if (new File(filename).exists()) {
+        if (new File(filename).canRead()) {
```

Comment:
Is preferable to split this control in two parts, for a better comprehension of the error

Reply:
I'm actually okay with the combined error here.  This should be an unusual occurrence and the error is verbose enough to permit troubleshooting.